src/commands/synthetics/run-test.ts

```diff
@@ -12,7 +12,7 @@ import {getReporter, getSuites, getTestsToTrigger, hasTestSucceeded, runTests, w
 export const DEFAULT_COMMAND_CONFIG: CommandConfig = {
   apiKey: '',
   appKey: '',
-  configPath: 'datadog-ci.json',
+  configPath: '',
```

Comment:
This breaks the behavior of trying to load `datadog-ci.json` by default (cf. [documentation](https://github.com/DataDog/datadog-ci/tree/master/src/commands/synthetics#configuration))

It should:
- try to load `datadog-ci.json`, if it does not exist, continue without error (neither log a message nor throw an exception)
- if a `--configPath` argument was passed, load this configured file, if it does not exist, throw an exception and stop the command